### ISSUE TITLE
Download multiple/all files/folders from cloud

### ISSUE THREAD

**Issue by fs1209** [TortugaPower/BookPlayer#1082]
## Feature request

Download multiple files or folders from cloud. Currently I can only download a single file or folder.

## Narrative

Currently I can only trigger the cloud-download for a single file or folder.

**GIVEN:**
- I've installed BookPlayer on a new device
- I'm logged into my account
- The Audiobook-List is synced and all files/folders have the cloud tag at the lower left corner
- I want do download all Audiobooks from the cloud
 
**WHEN:**
- Switch to multiple selection by tapping "Select"
- Select multiple or all Files/Folders
- Tap the "..." button "Download" function is disabled/grayed out
 
**THEN:**
- The "Download" button should be active
- All selected files/folders should be downloaded after tapping "Download"

## Design

Enable the Download option when multiple items are selected.
![image](https://github.com/TortugaPower/BookPlayer/assets/150681522/8d4efcc3-f2c9-4340-bdfd-7450373b0adb)

**Comment by GianniCarlo** [comment]
Good point @fs1209 , a workaround would be to instead download the Harry Potter folder from outside the folder. But before doing that, let me verify the issue with the library on your other ticket

**Comment by fs1209** [comment]
After I‘ve pressed the Submit button, I had the idea to put all of my files and folder into an enclosing folder, this way I should be able to download all books.It‘s still syncing, but I‘m really curious how it will work…Trifles make perfection, and perfection is no trifle.— MichelangeloAm 24.01.2024 um 14:29 schrieb Gianni Carlo ***@***.***>:﻿
Good point @fs1209 , a workaround would be to instead download the Harry Potter folder from outside the folder. But before doing that, let me verify the issue with the library on your other ticket
